lib/galaxy/jobs/runners/drmaa.py

```diff
@@ -310,11 +311,12 @@ def stop_job( self, job ):
         try:
             ext_id = job.get_job_runner_external_id()
             assert ext_id not in ( None, 'None' ), 'External job id is None'
-            if self.external_killJob_script is None:
+            kill_script = job.get_destination_configuration(self.app.config, "external_killJob_script", None)
```

Comment:
In Galaxy config it is called  `drmaa_external_killjob_script`. It is only called `external_killJob_script` in drmaa.py